Formula/c/ccextractor.rb

```diff
@@ -0,0 +1,41 @@
+class Ccextractor < Formula
+  desc "Tool for extracting closed captions from video files"
+  homepage "https://www.ccextractor.org"
+  url "https://github.com/CCExtractor/ccextractor/archive/refs/tags/v0.96.1.tar.gz"
+  sha256 "88579414ab2708b61e4787df752970c0cc3206bd455696db8a8843d46e55913c"
+  license "GPL-2.0-only"
+
+  depends_on "pkgconf" => :build
+  depends_on "rust" => :build
+  depends_on "freetype"
+  depends_on "gpac"
+  depends_on "libpng"
+  depends_on "protobuf-c"
+  depends_on "utf8proc"
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "llvm" => :build
+    depends_on "leptonica"
+    depends_on "tesseract"
+  end
+
+  def install
+    if OS.mac?
+      cd "mac" do
+        system "./build.command", "-system-libs"
+        bin.install "ccextractor"
+      end
+    else
+      cd "linux" do
+        ENV["LIBCLANG_PATH"] = Formula["llvm"].opt_lib.to_s
+        system "./build"
```

Comment:
Looks like there's no `-system-libs` option available in their Linux build script. I'll inquire upstream about this. As it currently stands, some system deps such as `libpng` are not used.